android/src/main/java/com/reactnativeadyendropin/ConfigurationParser.kt

```diff
@@ -70,6 +74,9 @@ class ConfigurationParser(private val clientKey: String, context: ReactApplicati
     val cardConfiguration = CardConfiguration.Builder(shopperLocale, environment, this.clientKey)
       .setShopperReference(shopperReference)
       .setSupportedCardTypes(CardType.MASTERCARD, CardType.VISA)
+      .setHolderNameRequired(getBoolean(config.getMap("card"), "showsHolderNameField"))
+      .setShowStorePaymentField(getBoolean(config.getMap("card"), "showsStorePaymentMethodField"))
+      .setHideCvc(!getBoolean(config.getMap("card"), "showsSecurityCodeField"))
```

Comment:
This overrides the default settings. Would be better to conditionally set these only if they exist in the config

Reply:
Good point. Updated now.